dynamiqs/utils/general.py

```diff
@@ -171,6 +172,53 @@ def sinm(x: QArrayLike) -> QArray:
     return -0.5j * (expm(1j * x) - expm(-1j * x))
 
 
+def signm(x: QArrayLike) -> QArray:
+    r"""Returns the operator sign function of a hermitian qarray.
```

Comment:
Is "operator sign function" the correct name mathematically? 🤔

Reply:
I'm not sure. How would you call it? I don't want to call it "sign function" only because it's ambiguous with the element-wise sign.